package.json

```diff
@@ -43,10 +43,10 @@
     "main.js"
   ],
   "dependencies": {
+    "@jridgewell/gen-mapping": "^0.3.0",
     "@jridgewell/trace-mapping": "^0.3.5",
     "acorn": "^8.5.0",
     "commander": "^2.20.0",
-    "source-map": "~0.8.0-beta.0",
```

Comment:
The tests still use this (namely test/mocha/sourcemaps.js) so it should go into devDependencies

Reply:
Ah, I didn't purge my `node_modules`. Fixed.